x/concentrated-liquidity/internal/swapstrategy/swap_strategy_test.go

```diff
@@ -39,29 +39,29 @@ func (suite *StrategyTestSuite) TestComputeSwapState() {
 	}{
 		"happy path: trade asset0 for asset1": {
 			sqrtPCurrent:    sdk.MustNewDecFromStr("70.710678118654752440"), // 5000
-			nextSqrtPrice:   sdk.MustNewDecFromStr("70.666662070529219856"), // 4993.777128190373086350
-			liquidity:       sdk.MustNewDecFromStr("1517818840.967515822610790519"),
+			nextSqrtPrice:   sdk.MustNewDecFromStr("70.666662070529219856"), // 4993.7771281903730
+			liquidity:       sdk.MustNewDecFromStr("1517882343.751510418088349649"),
```

Comment:
I'm just trusting these values as correct (I have no reference as old being true)

I'll review this more comprehensively in a later pass through CL (not this PR)